src/main/java/useCases/LogIn.java

```diff
@@ -13,14 +13,20 @@
 //A class to handle the Login functionality for different users
 public class LogIn {
 
+    private final String username;
+    private final String password;
+
+    public LogIn(String username, String password) {
+        this.username = username;
+        this.password = password;
+    }
 
     /**
-     * @param username: username of the User object to deserialize
```

Comment:
add parameters for each file including login

Reply:
doesn't cause any conflicts elsewhere, good!